src/index.html

```diff
@@ -22,7 +22,7 @@
   DEALINGS IN THE SOFTWARE.
 -->
 
-<html>
+<html spellcheck="false">
```

Comment:
@redmunds I think it's actually nicer to do keep it in the declarative markup. Why do it in JS if there's nothing dynamic about it?